### ISSUE TITLE
b81 - To Do List

### ISSUE THREAD

**Issue by mcsnetworks** [AlmuraDev/Almura#72]
The following needs to be done in B81 or 82.

Still needing to be done:
- Add custom tree generator
- Add permission to the coin-death drop

Add the following Crops:
- Fix Soybean image
- Add blueberries
- Add cranberries
- Add pineapples
- Add Sorghum
- Add Sugar Beets
- Add Hops

Add once custom Tree generator is finished:
- Add cherries

Things are already done:
- CoinExchange
- Forge Bridge Re-write
- Player Title fix
- Fix RayTrace issue
- Fix Collision Box issue
- Fix issue where custom blocks would attempt to kill player
- Fixed font scale issue within Almura Information Wand GUI
- Fixed custom crops getting broken by water dropping normal seed.
- Fixed Ambient Occlusion and lighting issues with custom blocks.
